src/extension.ts

```diff
@@ -253,6 +253,40 @@ export const checkConnection = (clearCookies = false): void => {
     });
 };
 
+async function serverManager(): Promise<void> {
+  const extId = "intersystems-community.servermanager";
+  const ignore =
+    config("ignoreInstallServerManager") ||
+    vscode.workspace.getConfiguration("intersystems.servers").get("/ignore", false);
+  if (ignore || vscode.extensions.getExtension(extId)) {
+    return;
+  }
+  return vscode.window
+    .showInformationMessage(
+      `Extension InterSystems® Server Manager have not installed,
+      it's recommended to install it.`,
+      "Install",
+      "Skip",
+      "Ignore"
+    )
+    .then(async (action) => {
+      switch (action) {
+        case "Install":
+          await vscode.commands.executeCommand("workbench.extensions.installExtension", extId);
+        // eslint-disable-next-line no-fallthrough
+        case "View":
```

Comment:
showInformationMessage doesn't offer "View" option

Reply:
My thoughts were, do not offer too many buttons, and was not sure if it's a good idea to install it from information message, not from extension page

Reply:
In which case why is `case "View":` here?